test/cmd/images-old-policy.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/../..
+source "${OS_ROOT}/hack/lib/init.sh"
+os::log::stacktrace::install
+trap os::test::junit::reconcile_output EXIT
+
+# Cleanup cluster resources created by this test
+(
+  set +e
+  oc login -u system:admin
+  oadm policy reconcile-cluster-roles --additive-only=false --confirm
+  exit 0
+) &>/dev/null
+
+
+os::test::junit::declare_suite_start "cmd/images-old-policy"
+
+oc project -q
+project=$(oc project -q)
```

Comment:
Interesting, when it isn't the correct project, do you see anything going on before the test begins that would indicate the `cmd-images-old-policy` namespace wasn't created correctly?